docs/setup.md

```diff
@@ -187,6 +187,26 @@ Alternatively you may submit an array of packages to define multiple versions:
 ]
 ```
 Once a custom package has been added to Private Packagist you will benefit from mirroring the zip file. Composer will have an additional location to download the file from if the original storage becomes unavailable.
+
+### Prevent a package from being used with custom packages
+
+While there is no direct "block package" feature in Packagist, you can effectively prevent a specific package from being used by creating a placeholder package. 
+
+The package type [metapackage](https://getcomposer.org/doc/04-schema.md#type) ensures no code is associated with it. Optionally, you can mark the package as [abandoned](https://getcomposer.org/doc/04-schema.md#abandoned) and suggest an alternative. 
```

Comment:
```suggestion
The package type [metapackage](https://getcomposer.org/doc/04-schema.md#type) ensures no code is associated with it. Optionally, you can mark the package as [abandoned](https://getcomposer.org/doc/04-schema.md#abandoned) and optionally suggest an alternative. 
```
They might not have a suggestion for an alternative package, if they just want to block something. It's all described in the docs you linked, but this might make it clearer that you don't need to suggest an alternative.

Reply:
Makes sense, thanks!